### PR TITLE
enable 'lein smoke' for CI

### PR DESCRIPTION
**Pull request by d-t-w**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -53,10 +62,12 @@
...
-  :eastwood {:add-linters [:unused-fn-args :unused-locals :unused-namespaces
+  :aliases {"smoke" ["do" ["modules" "puff"] ["clean"] ["check"] ["kibit"] ["cljfmt" "check"]]}
+
+  :eastwood {:add-linters [:unused-fn-args
+                           :unused-locals
+                           :unused-namespaces
                            :unused-private-vars]
              :namespaces  [:source-paths]}
 
-  :global-vars {*warn-on-reflection* true}
```
Comment (airwallex-james-sofra) [inline project.clj:60]: Does one of the linters handle this or are you not interested in doing it any longer?

**Review by airwallex-james-sofra — APPROVED** [review]
Looks good to me.

File: project.clj (hunk 1 above)
Comment (d-t-w) [reply project.clj:60]: I turned it on, and had heaps of warnings from 3rd-party libraries. 

Turned it off again and use 'lein check' specifically to target our own source. Not entirely sure I have it covered properly, need to take a look.

**Review by airwallex-james-sofra — APPROVED** [review]
This is approved.

**Review by jsofra — APPROVED** [review]
Once more for good luck.

**Pull request merged**